dist/rpm/openQA.spec

```diff
@@ -569,10 +566,6 @@ fi
 %config(noreplace) %attr(-,geekotest,root) %{_sysconfdir}/openqa/openqa.ini
```

Comment:
How about a comment in this file explaining how to use/override or point to the documentation